src/lwc/geWidgetService/widgetStubs.js

```diff
@@ -11,6 +11,18 @@ const ALLOCATION_WIDGET_DEVELOPER_NAME_STUBS = {
     ]
 }
 
+/*******************************************************************************
+* @description Stubs of object and field mapping developer names. Used to collect
+* the org specific mapping developer names for the widget geFormWidgetSoftCredits.
+*/
+const SOFT_CREDIT_WIDGET_DEVELOPER_NAME_STUBS = {
+    objectMappingDeveloperNameStubs: 'Opportunity_Contact_Role_1_',
+    fieldMappingDeveloperNameStubs: [
+        'Opportunity_Contact_Role_1_Role_',
+        'Opportunity_Contact_Role_1_Contact_'
```

Comment:
I believe these should be the following based on the default field mapping master labels and the BDI_ManageAdvancedMapping.generateRecordName() results.
```suggestion
        'Opportunity_Conct_Role_1_Role_',
        'Opportunity_ConRole_1_Contact_'
```